### PR TITLE
Refactoring classes into packages

### PR DESCRIPTION
**Pull request by Averynder**
This branch refactors the previously committed code and classes into the appropriate packages. None of the code was changed besides for the import statement and any package names associated with the changes. This closes issue #178

### CODE REVIEW

File: app/src/main/java/com/example/scoutconcordia/DataStructures/Graph.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.example.scoutconcordia;
+package DataStructures;
```
Comment (vincentcerri) [inline app/src/main/java/com/example/scoutconcordia/DataStructures/Graph.java]: should be : package com.example.scoutconcordia.DataStructures;

File: app/src/main/java/com/example/scoutconcordia/DataStructures/LinkedList.java [hunk 2]
```diff
@@ -1,14 +1,6 @@
-package com.example.scoutconcordia;
+package DataStructures;
```
Comment (vincentcerri) [inline app/src/main/java/com/example/scoutconcordia/DataStructures/LinkedList.java]: should be: package com.example.scoutconcordia.DataStructures;

File: app/src/main/java/com/example/scoutconcordia/FileAccess/DES.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.example.scoutconcordia;
+package FileAccess;
```
Comment (vincentcerri) [inline app/src/main/java/com/example/scoutconcordia/FileAccess/DES.java]: should be: package com.example.scoutconcordia.FileAccess;

File: app/src/main/java/com/example/scoutconcordia/MapInfoClasses/BuildingInfo.java [hunk 4]
```diff
@@ -1,17 +1,15 @@
-package com.example.scoutconcordia;
+package MapInfoClasses;
```
Comment (vincentcerri) [inline app/src/main/java/com/example/scoutconcordia/MapInfoClasses/BuildingInfo.java]: should be: package com.example.scoutconcordia.MapInfoClasses;

File: app/src/main/java/com/example/scoutconcordia/MapInfoClasses/CustomInfoWindow.java [hunk 5]
```diff
@@ -1,13 +1,16 @@
-package com.example.scoutconcordia;
+package MapInfoClasses;
```
Comment (vincentcerri) [inline app/src/main/java/com/example/scoutconcordia/MapInfoClasses/CustomInfoWindow.java]: should be: package com.example.scoutconcordia.MapInfoClasses;

File: app/src/main/java/com/example/scoutconcordia/DataStructures/Graph.java (hunk 1 above)
Comment (Averynder) [reply app/src/main/java/com/example/scoutconcordia/DataStructures/Graph.java]: When a class is in the same package Java imports the items automatically unless you want Node to be a separate class though it is only intended for use with the Graph. The naming has been fixed.

File: app/src/main/java/com/example/scoutconcordia/FileAccess/DES.java (hunk 3 above)
Comment (Averynder) [reply app/src/main/java/com/example/scoutconcordia/FileAccess/DES.java]: Fixed

File: app/src/main/java/com/example/scoutconcordia/MapInfoClasses/CustomInfoWindow.java (hunk 5 above)
Comment (Averynder) [reply app/src/main/java/com/example/scoutconcordia/MapInfoClasses/CustomInfoWindow.java]: Fixed

**Pull request merged**